### PR TITLE
Add note about PNG files to Image static resources

### PR DESCRIPTION
**Pull request by andersryanc**

### CODE REVIEW

**Comment by ide** [comment]
@kmagiera can you make sure this is accurate?

**Comment by kmagiera** [comment]
Not sure if that's a correct workaround, maybe @frantic will know as he was commenting on the original issue

**Comment by frantic** [comment]
We are working on a better asset system that will hopefully solve this problem for good. Meanwhile I'm okay with adding this note, however:

1. You probably don't need `isStatic` attribute.
2. After renaming `require('image!..)` syntax still doesn't work?

File: docs/Image.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ When your entire codebase respects this convention, you're able to do interestin
 
 *This process is currently being improved, a much better workflow will be available shortly.*
 
+> **NOTE**: PNG images are required when loading with `require('image!my-icon')`
+>
+> At this time, only PNG images are supported in iOS. There is an [issue](https://github.com/facebook/react-native/issues/646) that is currently addressing this bug. In the meantime a quick fix is to rename your files to image.png or to use the `isStatic` flag like: `source={{ uri: 'image', isStatic: true }}`.
```
Comment (frantic) [inline docs/Image.md]: nit: please be consistent with your examples, in this case change `uri` to `my-icon`

**Comment by andersryanc** [comment]
1. You're right, `isStatic` is not necessary.
2. I'm pretty sure that is what I did at first (and it was working), but for some reason, my tests now seem to be producing an error with `require()`: `Requiring unknown module "image!my-icon"...`

**Comment by andersryanc** [comment]
I figured it out, I was doing the "bad" example and trying to load an array of images with:

```
var PREF_IMAGES = [
  // 'beer', 'breakfast', 'coffee', 'cycling', 'ice-cream', 'pie',
  'snowboarding', 'steak', 'sushi', 'tea', 'veggies', 'wine'
];

var Thumb = React.createClass({
  shouldComponentUpdate: function(nextProps, nextState) {
    return false;
  },
  render: function() {
    return (
      <View style={styles.row}>
        <Image style={styles.image} source={require('image!pref-' + this.props.uri)} />
        <Text style={styles.imageCaption}> {this.props.uri} </Text>
      </View>
    );
  }
});
var createThumbRow = (uri, i) => <Thumb key={i} uri={uri} />;
```

Then in my main component:

```
// ...
<View>
        {PREF_IMAGES.map(createThumbRow)}
</View>
// ...
```

This works fine when using the `source={{ uri: 'pref-' + this.props.uri }}` method.

**Pull request merged**